docs/extensions/tutorials/_posts/1970-01-10-DebugSettingsPages.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/debug-settings-pages
+title: Debugging and local development of settings pages
+section: Tutorials
+---
+
+# Debugging and Local Development
+<hr />
+
+This short tutorial covers how to improve your settings page development flow by utilizing the `Debug` feature in the Builder. It'll explain how to see code changes without having to `push` every code change and how to use dev tools to debug the settings pages through the Builder.
+
+> #### Note
+> HTML settings pages currently cannot use the `Debug` feature in the Builder.
+
+## Local development
+
+When you start developing settings pages, you'll quickly realize how much of a hassle it is to `shoutem publish` every change to see the updates on the Builder. To avoid this and enjoy a much more efficient development flow, you can simply do the following:
```

Comment:
When you start developing settings pages, you'll quickly realize that it's inpractical to `shoutem publish` every change to Shoutem just to see the updated settings page.